### PR TITLE
feat(gallery): update template accordingly to ECL 2.8.0 - INNO-1626

### PR DESCRIPTION
**Pull request by aritomelo**
# PR description

Update gallery twig template.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] I have put the vanilla component as `devDependencies`
- [ ] I have put the specs package as `devDependencies`
- [ ] I have added the components directly used in the twig file (with `include` or `embed`) as `dependencies`
- [ ] My component is listed in `@ecl-twig/ec-components`'s `dependencies`
- [ ] My variables naming follow the guidelines (snake case for twig)
- [ ] I have provided tests
- [ ] I have provided documentation (for the "notes" tab)
- [ ] If my local `yarn.lock` contains changes, I have committed it
- [ ] I have given my PR the proper label (`pr: review needed` to indicate that I'm done and now waiting for a review ,`pr: wip` to indicate that I'm actively working on it ...)

### CODE REVIEW

File: src/ec/packages/ec-component-gallery/__snapshots__/gallery.test.js.snap [hunk 1]
```diff
@@ -261,1134 +262,28 @@ exports[`EC - Gallery Default renders correctly 1`] = `
       class="ecl-gallery__item"
     >
       <a
-        aria-label="VIDEO NOT SUPPORTED YET"
```
Comment (aritomelo) [inline src/ec/packages/ec-component-gallery/__snapshots__/gallery.test.js.snap]: @yhuard Should we keep this sentence "VIDEO NOT SUPPORTED YET" ?

**Comment by aritomelo** [comment]
@emeryro OK, I'll.

File: src/ec/packages/ec-component-button/button.html.twig [hunk 2]
```diff
@@ -25,7 +25,7 @@
 {% set _extra_attributes = '' %}
 {% set _label = label|default('') %}
 {% set _variant = variant|default('primary') %}
-{% set _type = type|default('submit') %}
+{% set _type = type|default('button') %}
```
Comment (emeryro) [inline src/ec/packages/ec-component-button/button.html.twig]: the default should stay as submit, and be overrided when needed.
here it changed it for someother components too (search form, site header).

Comment (aritomelo) [reply src/ec/packages/ec-component-button/button.html.twig]: ok.

Comment (aritomelo) [reply src/ec/packages/ec-component-button/button.html.twig]: It is solved @emeryro

**Pull request merged**